dll/cpl/inetcpl/lang/fr-FR.rc

```diff
@@ -27,15 +27,15 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_CPL_NAME        "Internet Settings"
-    IDS_CPL_INFO        "Configure WINE Internet Explorer and related settings."
-    IDS_SEC_SETTINGS    "Security settings for zone: "
-    IDS_SEC_LEVEL0      "Custom"
-    IDS_SEC_LEVEL1      "Very Low"
-    IDS_SEC_LEVEL2      "Low"
-    IDS_SEC_LEVEL3      "Medium"
-    IDS_SEC_LEVEL4      "Increased"
-    IDS_SEC_LEVEL5      "High"
+    IDS_CPL_NAME        "Paramètres Internet"
```

Comment:
This commit is #2571. Remove it from here.

Reply:
How can I remove inetcpl file from the PR ?

Reply:
I already told you on Mattermost: `git fetch --all && git rebase upstream/master && git push --force-with-lease` to be run when you are in your local branch. (Here "upstream" is the name of the reactos/reactos repo)

Reply:
So I have to do this then change again and push again ?